### DISCUSSION TITLE
Uptimes for proget.makedeb.org

### DISCUSSION THREAD

**Discussion by devesh6s** [makedeb/prebuilt-mpr#703] [Issues]
proget.makedeb.org CNAMED to proxy.hunterwittenborn.com hosted on linode is down. This is causing failures on our github codespaces devcontainers that rely heavily on prebuilt binaries available on https://proget.makedeb.org/debian-feeds/prebuilt-mpr.pub

@hwittenborn let us know if the community can help with the infrastructure / donate to extend the uptimes.

**Comment by hwittenborn** [comment]
Hey @devesh6s, sorry for the outage there. There were some issues from a restart on the server the Prebuilt-MPR is hosted on, but everything should be good to go now.

I don't have any major issues with makedeb's current infrastructure, but my main constraint at the moment is definitely with available time due to financial reasons that hinder me from fixing things when they go wrong.

I can't say a ton about my situation at the moment (nothing serious, just some things with my available time), but I plan on making a public announcement about everything soon. I'll let you know once I have a more concrete answer I can provide.

**Reply by devesh6s** [reply]
Appreciated everything you did and thank you for quick resolution. Happy to help with infrastructure or code maintenance.
